server/storage/schema/schema.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	"fmt"
+
+	"github.com/coreos/go-semver/semver"
+	"go.uber.org/zap"
+
+	"go.etcd.io/etcd/server/v3/storage/backend"
+)
+
+var (
+	V3_5 = semver.Version{Major: 3, Minor: 5}
```

Comment:
Since we already have these and to me they seem more like constants, what do you think we take them from https://github.com/etcd-io/etcd/blob/f4fad92e0d0240d1495cc6bf9d2f4d1fbeab4e4b/server/etcdserver/version/version.go#L28-L29